extra/src/main/scala/ai/lum/odinson/extra/IndexDocuments.scala

```diff
@@ -17,7 +17,7 @@ import ai.lum.common.Serializer
 import ai.lum.labrador.DocumentMetadata
 import ai.lum.odinson.lucene.analysis._
 import ai.lum.odinson.OdinsonIndexWriter
-
+import java.nio.charset.StandardCharsets.UTF_8
```

Comment:
Why is it needed? I don't see it being used anywhere in this file and it doesn't look like it's a scala implicit definition

Reply:
Right, seems to be redundant, I removed it.